diff --git a/CMakeLists.txt b/CMakeLists.txt
index 63a17b778e4..5737deb01f8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -223,11 +223,6 @@ install(
     COMPONENT runtime)
 
 configure_file(bin/rocprofv2 ${PROJECT_BINARY_DIR} COPYONLY)
-install(
-    FILES ${PROJECT_SOURCE_DIR}/bin/rocprofv2
-    DESTINATION ${CMAKE_INSTALL_BINDIR}
-    PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-    COMPONENT runtime)
 
 install(
     FILES ${CMAKE_CURRENT_SOURCE_DIR}/bin/txt2xml.sh
